### PR TITLE
ChatroomModel

### PR DESCRIPTION
**Pull request by LightspeedDude**
Added Chatroom Model

### CODE REVIEW

**Comment by LightspeedDude** [comment]
Added ChatroomModel

File: simple-chat-back/src/models/chatroomModel.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+const mongoose = require('mongoose');
```
Comment (rem1niscence) [inline simple-chat-back/src/models/chatroomModel.js]: remove semicolon, @Lunaticfrg0  configured the project to not use them

File: simple-chat-back/src/models/chatroomModel.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    text: {
+        type: mongoose.Schema.Types.ObjectId,
+        ref: "Text",
+        required: true,
+        trim: true
+      }
+})
+
+const Chatroom = mongoose.model('Chatroom', chatroomSchema )
+module.exports = Chatroom
```
Comment (rem1niscence) [inline simple-chat-back/src/models/chatroomModel.js:25]: add an empty newline. As a rule of thumb, all files should always end with an empty newline

File: simple-chat-back/src/models/chatroomModel.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    },
+    text: {
+        type: mongoose.Schema.Types.ObjectId,
+        ref: "Text",
+        required: true,
+        trim: true
+      }
+})
+
+const Chatroom = mongoose.model('Chatroom', chatroomSchema )
```
Comment (rem1niscence) [inline simple-chat-back/src/models/chatroomModel.js]: remove space before closing parentheses

**Review by rem1niscence — CHANGES_REQUESTED** [review]
Your commit message should explain in a sentence what you did on the code. (i.e: Add Chatroom Mongoose Model)

**Review by rem1niscence — APPROVED** [review]

**Pull request merged**